amgut/lib/data_access/ag_data_access.py

```diff
@@ -323,8 +340,11 @@ def deleteAGParticipantSurvey(self, ag_login_id, participant_name):
                 self.deleteSample(info['barcode'], ag_login_id)
 
             # Delete last due to foreign keys
-            sql = "DELETE FROM ag_kit_barcodes WHERE survey_id IN %s"
+            sql = """DELETE FROM ag.source_barcodes_surveys
+                     WHERE survey_id IN %s"""
             TRN.add(sql, [tuple(survey_ids)])
+            sql = "DELETE FROM ag.ag_kit_barcodes WHERE barcode IN %s"
```

Comment:
oh. I thought the whole ag_login_id should be removed (in the doc '# Remove user from new schema'), but you might be right that we only want to remove the "source", i.e. the combination of ag_login_id and participant_name. 
It should be sufficient to just remove the relation in source_barcodes_surveys, right?